extra/webapps/imagebin/upload-image.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version='1.0' ?>
+<!DOCTYPE html>
 <html>
```

Comment:
Now I'm not sure about removing this line (same for uploded-image.xml) Maybe it should be added back ? Sorry for the noise